apps/transport/lib/transport/import_data.ex

```diff
@@ -261,16 +261,10 @@ defmodule Transport.ImportData do
         ]
       }) do
     # For the EPCI we get the list of cities contained by the EPCI
-    EPCI
-    |> Repo.get_by(code: code)
-    |> case do
-      nil ->
-        Logger.warning("impossible to find epci #{code}, no cities associated to the dataset")
-        []
-
-      epci ->
-        epci.communes_insee
-    end
+    Commune
```

Comment:
Ici je suis pas méga serein, à part le test (plus bas) j’ai pas testé en local avec de la vraie data. Il faut modifier les variables de développement pour pointer vers data.gouv.fr, ou avec demo.data.gouv.fr ça passe ?

Reply:
`demo.` est pas toujours synchro avec la production donc tu risques d'avoir des JDDs qui ne sont pas trouvés.

Tu peux tester en local au moins l'import d'un JDD depuis le BO pour lequel on a un EPCI associé sur data.gouv.fr (les JDDs où `associated_territory_name = 'Métropole de Lyon'` il me semble) en vérifiant avant que la réponse de l'API de `demo.` est bonne pour le champ `spatial`

Reply:
Alors, la réponse est bonne, ça marche, mais je suis assez surpris par le nombre de requêtes SQL :

```
[info] getting zone (url = https://demo.data.gouv.fr/api/1/spatial/zones/fr:epci:200046977/)
[debug] QUERY OK source="commune" db=0.6ms queue=0.7ms idle=1389.3ms
SELECT c0."insee" FROM "commune" AS c0 WHERE (c0."epci_insee" = $1) ["200046977"]
[debug] QUERY OK source="dataset" db=0.8ms queue=1.5ms idle=1392.9ms
SELECT d0."id", d0."datagouv_id", d0."custom_title", d0."created_at", d0."description", d0."frequency", d0."last_update", d0."licence", d0."logo", d0."full_logo", d0."slug", d0."tags", d0."datagouv_title", d0."type", d0."organization", d0."organization_type", d0."has_realtime", d0."is_active", d0."population", d0."nb_reuses", d0."latest_data_gouv_comment_timestamp", d0."archived_at", d0."custom_tags", d0."inserted_at", d0."updated_at", d0."legal_owner_company_siren", d0."region_id", d0."aom_id", d0."organization_id", d0."associated_territory_name" FROM "dataset" AS d0 WHERE (d0."datagouv_id" = $1) ["5e3aecac634f417a73b726e5"]
[debug] QUERY OK source="region" db=0.5ms queue=0.6ms idle=931.9ms
SELECT r0."id", r0."nom", r0."insee", r0."is_completed", r0."geom", d1."dataset_id"::bigint FROM "region" AS r0 INNER JOIN "dataset_region_legal_owner" AS d1 ON r0."id" = d1."region_id" WHERE (d1."dataset_id" = ANY($1)) ORDER BY d1."dataset_id"::bigint [[755]]
[debug] QUERY OK source="aom" db=0.7ms queue=0.5ms idle=519.0ms
SELECT a0."id", a0."composition_res_id", a0."insee_commune_principale", a0."siren", a0."nom", a0."forme_juridique", a0."nombre_communes", a0."population", a0."surface", a0."geom", a0."region_id", a0."departement", d1."dataset_id"::bigint FROM "aom" AS a0 INNER JOIN "dataset_aom_legal_owner" AS d1 ON a0."id" = d1."aom_id" WHERE (d1."dataset_id" = ANY($1)) ORDER BY d1."dataset_id"::bigint [[755]]
[debug] QUERY OK source="resource" db=0.2ms queue=0.2ms idle=459.8ms
SELECT r0."id", r0."url", r0."format", r0."last_import", r0."title", r0."last_update", r0."latest_url", r0."is_available", r0."is_community_resource", r0."schema_name", r0."schema_version", r0."community_resource_publisher", r0."description", r0."original_resource_url", r0."datagouv_id", r0."filesize", r0."filetype", r0."type", r0."display_position", r0."dataset_id", r0."dataset_id" FROM "resource" AS r0 WHERE (r0."dataset_id" = $1) ORDER BY r0."dataset_id" [755]
[debug] QUERY OK source="organization" db=0.3ms queue=0.2ms idle=319.0ms
SELECT o0."id", o0."slug", o0."name", o0."acronym", o0."logo", o0."logo_thumbnail", o0."badges", o0."metrics", o0."created_at", o0."id" FROM "organization" AS o0 WHERE (o0."id" = $1) ["534fff7aa3a7292c64a77e0c"]
[debug] QUERY OK source="commune" db=11.0ms queue=0.3ms idle=261.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee", d1."dataset_id"::bigint FROM "commune" AS c0 INNER JOIN "dataset_communes" AS d1 ON c0."id" = d1."commune_id" WHERE (d1."dataset_id" = ANY($1)) ORDER BY d1."dataset_id"::bigint [[755]]
[debug] QUERY OK source="commune" db=5.7ms queue=0.4ms idle=154.7ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69003"]
[debug] QUERY OK source="commune" db=5.5ms idle=83.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69029"]
[debug] QUERY OK source="commune" db=4.9ms idle=70.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69033"]
[debug] QUERY OK source="commune" db=5.2ms idle=49.0ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69034"]
[debug] QUERY OK source="commune" db=5.7ms idle=49.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69040"]
[debug] QUERY OK source="commune" db=5.4ms idle=48.6ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69044"]
[debug] QUERY OK source="commune" db=5.1ms idle=54.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69046"]
[debug] QUERY OK source="commune" db=6.1ms idle=55.7ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69063"]
[debug] QUERY OK source="commune" db=4.5ms idle=62.0ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69068"]
[debug] QUERY OK source="commune" db=0.2ms idle=44.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69069"]
[debug] QUERY OK source="commune" db=0.2ms idle=39.5ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69071"]
[debug] QUERY OK source="commune" db=0.2ms idle=34.7ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69072"]
[debug] QUERY OK source="commune" db=0.1ms idle=29.4ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69081"]
[debug] QUERY OK source="commune" db=0.1ms idle=23.6ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69085"]
[debug] QUERY OK source="commune" db=0.1ms idle=18.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69087"]
[debug] QUERY OK source="commune" db=0.1ms idle=13.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69088"]
[debug] QUERY OK source="commune" db=0.1ms idle=6.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69089"]
[debug] QUERY OK source="commune" db=0.2ms idle=5.0ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69091"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.7ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69096"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.5ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69100"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69116"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69117"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69123"]
[debug] QUERY OK source="commune" db=0.2ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69127"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69142"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69143"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69149"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69152"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69153"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69163"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69191"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69194"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69199"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69202"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69204"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69205"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.0ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69207"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69233"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69244"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69250"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69256"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69259"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69260"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69266"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69273"]
[debug] QUERY OK source="commune" db=0.2ms idle=1.8ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69275"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69276"]
[debug] QUERY OK source="commune" db=0.1ms idle=1.9ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69278"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.0ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69279"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69282"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69283"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.1ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69284"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69286"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.4ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69290"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.5ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69292"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.4ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69293"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.3ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69296"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69271"]
[debug] QUERY OK source="commune" db=0.1ms idle=2.2ms
SELECT c0."id", c0."insee", c0."nom", c0."surf_ha", c0."geom", c0."population", c0."siren", c0."arrondissement_insee", c0."aom_res_id", c0."region_id", c0."departement_insee", c0."epci_insee" FROM "commune" AS c0 WHERE (c0."insee" = $1) ["69168"]
[debug] QUERY OK source="organization" db=0.3ms queue=0.2ms idle=4.7ms
SELECT o0."id", o0."slug", o0."name", o0."acronym", o0."logo", o0."logo_thumbnail", o0."badges", o0."metrics", o0."created_at" FROM "organization" AS o0 WHERE (o0."id" = $1) ["534fff7aa3a7292c64a77e0c"]
```

Obtenu en allant sur http://localhost:5000/backoffice/datasets/755/edit puis en cliquant sur «importer».

En fait j’ai l’impression que c’était déjà comme ça et pas dû à mon code : la seule requête dûe à mon changement c’est celle-ci :
```SELECT c0."insee" FROM "commune" AS c0 WHERE (c0."epci_insee" = $1) ["200046977"]
[debug] QUERY OK source="dataset" db=0.8ms queue=1.5ms idle=1392.9ms```

Toutes les autres devaient déjà être là. Je vais vérifier sur master au cas où.

Reply:
Peut-être prévoir un petit refactor pour un `where in` ? Dans cette PR ou plus tard, your choice

Reply:
Veux-tu le traiter dans cette PR ou souhaites-tu le noter à faire dans une issue ?

Reply:
Issue ici https://github.com/etalab/transport-site/issues/3707 mais je vais la traiter de suite.